scripts/includes/common.sh

```diff
@@ -90,7 +90,7 @@ check_st2_host_dependencies() {
 
   for i in "${ports[@]}"
   do
-    rv=$(port_status $i)
+    rv=$(port_status $i | sed 's/.*-$\|.*systemd\|.*beam.smp.*\|.*epmd\|.*st2.*\|.*nginx.*\|.*python.*\|.*postgres\|.*postmaster.*\|.*mongod//')
```

Comment:
Why the rule `.*-$` is needed?
Can you provide any example matching that regex?

Reply:
I receive the following running

```
sudo netstat -tunlp --inet | awk -v port=:9100 '$4 ~ port && $6 ~ /LISTEN/ { print $4 " " $7 }'
```

on each of the platforms:

u14: `127.0.0.1:9100 15018/python`
u16: `127.0.0.1:9100 1/systemd`
el6: `127.0.0.1:9100 12283/python`
el7: `127.0.0.1:9100 1/systemd`
st2-docker u14: `127.0.0.1:9100 -`

Reply:
I need the `.*-$` pattern when running in st2-docker. I suppose it can be simplified to `-$`.

Reply:
This is from our AWS infra, Ubuntu16 AMI: 
```
arma@test001:~$ sudo netstat -tunlp --inet | awk -v port=:9100 '$4 ~ port && $6 ~ /LISTEN/ { print $4 " " $7 }'
127.0.0.1:9100 1/init
```

So merging output from our both environments, adding `1/init` along with `1/systemd` would be good.